src/wp-includes/global-styles-and-settings.php

```diff
@@ -255,3 +255,70 @@ function ( $item ) {
 		}
 	}
 }
+
+/**
+ * Whether a theme or its parent have a theme.json file.
+ *
+ * @since 6.2.0
+ *
+ * @return boolean
+ */
+function wp_theme_has_theme_json() {
+	/**
```

Comment:
```suggestion
	/*
```
Multi-line comments use `/*` rather than `/**` (which is for docblocks + filter comments). See [here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments).